public/js/pages/install-app.js

```diff
@@ -0,0 +1,83 @@
+requirejs.config({
+  baseDir: "/js",
+  paths: {
+    "analytics": "/bower_components/webmaker-analytics/analytics",
+    "jquery": "/bower_components/jquery/jquery.min",
+    "intl-tel-input": "/bower_components/intl-tel-input/build/js/intlTelInput.min"
+  }
+});
+
+require(["jquery", "analytics", "intl-tel-input"], function ($, analytics) {
+  "use strict";
+  var downloadButtons = $(".download-app-btn"),
+    sendInstallAppButtonContainer = $(".desktop-install-app-btn-wrapper"),
+    sendInstallLinkContainer = $(".send-install-link-container"),
+    phoneNumberLabel = $(".phone-number-label"),
+    getAppBtn = $(".desktop-install-app-btn"),
+    phoneNumberInput = $(".phone-num-input"),
+    sendSmsButton = $(".send-sms-btn"),
+    messageSent = $(".message-sent-container"),
+    messageSentError = $(".message-sent-error-container"),
+    csrfToken = $("meta[name='csrf-token']").attr("content");
+
+  function sendSMS() {
+    sendSmsButton.attr("disabled", true);
+    analytics.event("Sending SMS Link");
```

Comment:
@adamlofting I'm going to add a `analytics.event("Click Get App Button");` event for when the user clicks to open the send SMS form. Does that event text sound good?